addons/im_livechat/controllers/main.py

```diff
@@ -128,7 +129,7 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
         if previous_operator_id:
             previous_operator_id = int(previous_operator_id)
 
-        chatbot_script = False
+        chatbot_script = request.env['chatbot.script']
```

Comment:
We try to apply ruff formatting in new diff in our modules (in particular it would be double quotes in this case)